versioned_docs/version-1.0/cadence-migration-guide/usdc-migration.mdx

```diff
@@ -0,0 +1,330 @@
+and mint the exact same amount of `USDCFlow`
+using the public `USDCFlow.wrapFiatToken()` function
+
+During the Crescendo migration, the `USDCFlow` contract will become part of the
+Flow VM bridge protocol and become the bridged version of USDC on Flow EVM.
+This USDC will be backed by real USDC from Ethereum mainnet, ensuring that 
+`USDCFlow` will seamlessly retain its 1 USD value and be able to be redeemed
+with Circle by bridging back to Ethereum mainnet.
+
+This document is primarily focused on what developers who rely on `FiatToken`
+can do to migrate to `USDCFlow` with as little disruption as possible.
+A brief guide is provided in the next section for regular USDC users
+about what they can do to migrate to the new token, but those users should primarily
+follow the [Flow Blog](https://www.flow.com/authors/flow) and communication
+from the apps they use about what their options are to migrate to the new USDC on Flow.
+
+# What can regular USDC holders do?
+
+If you are a user with USDC in your account, the recommended path is to directly 
+swap your USDC with `USDCFlow` using this transaction. This functionality
+is available on [Flow Port](https://port.flow.com/) with any Flow-compatible wallet.
+
+As described in the blog post linked in the previous section, USDC holders
+can also swap to FLOW on [increment.fi](https://app.increment.fi/),
+deposit to [Dapper Wallet](https://meetdapper.com/) as Dapper balance, 
+or wait until after September 3rd and manually redeem with Circle.
+
+If you are a USDC holder on Flow and you use an app that transacts in USDC in any way,
+you should reach out to the app's support channels to see if there is anything specific
+you need to do to migrate your usage of that app to the new `USDCFlow` token.
+The Flow foundation is already working with most apps that use USDC
+to help them with their migration plans to the new `USDCFlow`,
+so rest assured that there will be a path for you to retain your assets
+past the Crescendo migration, regardless of what form that takes.
+
+If you cannot get in touch with the support channels of whatever app you use,
+please reach out in the [Flow Discord](https://discord.com/invite/J6fFnh2xx6)
+with your issues and the Flow team will definitely help you get it resolved.
+
+## Flow Users with Marketplace listings for USDC
+
+If you are a user with a NFT listing on a marketplace with USDC as the payment currency,
+your existing listings will not work any more after the Crescendo upgrade.
+If you still want your listings to work, you will need to create new listings
+with the new `USDCFlow` currency so they are still available after the Crescendo upgrade.
+This will likely require that whatever app you were using for marketplace listings
+supports the transaction that lists for `USDCFlow` instead of `FiatToken`,
+so if this isn't immediately available, please contact their support
+to ask if it will be supported.
```

Comment:
```suggestion
This will likely require the marketplace app you used for your listings to support the transaction for listing with `USDCFlow` instead of `FiatToken`. If this functionality is not immediately available, please contact the app's support team to inquire about their plans to add support for `USDCFlow` listings.
```